anvil_schema/fss_index_data_dictionary/fss_index_data_dictionary.json

```diff
@@ -0,0 +1,888 @@
+      ],
+      "primaryKey": ["activity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_alignmentactivity",
+      /// ???
+      "description": "Activity table designed to capture information about the alignment process in next generation sequencing.",
+      "columns": [
+        {
+          "name": "alignmentactivity_id", 
+          /// ????
+          "description": "Unique ID used to identify the alignment instance represented by this record.",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": true
+        },
+        {
+          "name": "activity_type", 
+          "description": "A reference to the type of activity, preferably using an identifier in a recommended standard ontology.",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "data_modality", 
+          "description": "Data modality describes the biological nature of the information gathered as the result of an Activity, independent of the technology or methods used to produce the information.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "generated_file_id", 
+          "description": "Unique ID used to identify the file or files generated by an activity.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "used_file_id", 
+          "description": "Unique ID used to identify the file or files used within an activity.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "reference_assembly", 
+          "description": "A reference to the collection of sequences taken as the standard for a given organism. May be defined by https://www.ncbi.nlm.nih.gov/grc.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "source_datarepo_row_ids", 
+          "description": "The identifiers of the Terra Data Repository rows that contributed to the creation of the record. For example, if row 123 in source table 'subject' was used to create row ABC in the 'anvil_donor' table, the source_datarepo_row_ids column for row ABC in 'anvil_donor' would contain 'subject:123'.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false}
+      ],
+      "primaryKey": ["alignmentactivity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_antibody",
+      "description": "May be a reference to an antibody used for a SampleTreatmentActivity or another Activity.",
```

Comment:
"SampleTreatmentActivity" --> "sample treatment activity"

"Activity" --> "activity"

"May be a reference..." --> "A reference to an..."